templates/tickets/ticket.html

```diff
@@ -21,10 +21,16 @@ <h3>Ticket details</h3>
                  {% endif %}</li>
             <li><em>assigned to</em>
                  {% if ticket.assignee %}
-                     {{ ticket.assignee.username }}
+                     <a href="{% url "account" ticket.assignee.username %}">{{ ticket.assignee.username }}</a>
                  {% else %}
                      no one
-                 {% endif %}</li>
+                 {% endif %}
+
+                 {# Show 'assign to me' link if the assigned moderator is not you #}
+                 {% if ticket.assignee.id != user.id %}
+                 (<a href="{% url 'tickets-moderation-assign-single-ticket' ticket.sender.id ticket.id %}?next=ticket">assign to me</a>)
```

Comment:
Maybe we could also add the green cross here as it is always used in "assign to me" actions.

Reply:
It makes the UI look ugly without further adjustments, I would prefer not putting it there. Anyway as I understand it will be redesigned with the new frontend.
![screenshot_20180323_152817](https://user-images.githubusercontent.com/1110902/37835319-2f7c05ea-2eb0-11e8-8911-789439fd933c.png)

Reply:
ok, fair enough